src/entities/createPool/composableStable/createPoolComposableStable.ts

```diff
@@ -46,9 +46,9 @@ export class CreatePoolComposableStable implements CreatePoolBase {
             input.name || input.symbol,
             input.symbol,
             tokens,
-            input.amplificationParameter,
+            BigInt(input.amplificationParameter),
```

Comment:
Why don't we expect the user to provide the proper types? Seems to make more sense than casting it here.